BM/instruction-check/feature_list.py

```diff
@@ -1,51 +1,228 @@
+        "cpuid": ['7', '0', '0', '0', 'c', '6'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "CET_SS": {
+        "cpuid": ['7', '0', '0', '0', 'c', '7'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "GFNI": {
+        "cpuid": ['7', '0', '0', '0', 'c', '8'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "VAES": {
+        "cpuid": ['7', '0', '0', '0', 'c', '9'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "AVX512_VNNI": {
+        "cpuid": ['7', '0', '0', '0', 'c', '11'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "AVX512_BITALG": {
+        "cpuid": ['7', '0', '0', '0', 'c', '12'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "RDPID": {
+        "cpuid": ['7', '0', '0', '0', 'c', '22'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "KL": {
+        "cpuid": ['7', '0', '0', '0', 'c', '23'],
+        "platforms": {}
+    },
+    "CLDEMOTE": {
+        "cpuid": ['7', '0', '0', '0', 'c', '25'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "MOVDIRI": {
+        "cpuid": ['7', '0', '0', '0', 'c', '27'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "MOVDIR64B": {
+        "cpuid": ['7', '0', '0', '0', 'c', '28'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "PKS": {
+        "cpuid": ['7', '0', '0', '0', 'c', '31'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "UINTR": {
+        "cpuid": ['7', '0', '0', '0', 'd', '5'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "AVX512_VP2INTERSECT": {
+        "cpuid": ['7', '0', '0', '0', 'd', '8'],
+        "platforms": {}
+    },
+    "SERIALIZE": {
+        "cpuid": ['7', '0', '0', '0', 'd', '14'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "TSXLDTRK": {
+        "cpuid": ['7', '0', '0', '0', 'd', '16'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "CET_IBT": {
+        "cpuid": ['7', '0', '0', '0', 'd', '20'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "AMX_BF16": {
+        "cpuid": ['7', '0', '0', '0', 'd', '22'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "AVX512_FP16": {
+        "cpuid": ['7', '0', '0', '0', 'd', '23'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "AMX_TILE": {
+        "cpuid": ['7', '0', '0', '0', 'd', '24'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "AMX_INT8": {
+        "cpuid": ['7', '0', '0', '0', 'd', '25'],
+        "platforms": {"SPR", "EMR", "GNR"}
+    },
+    "AVX_VNNI": {
+        "cpuid": ['7', '0', '1', '0', 'a', '4'],
+        "platforms": {"SPR", "EMR", "GNR", "SRF", "CWF"}
+    },
+    "CMPCCXADD": {
+        "cpuid": ['7', '0', '1', '0', 'a', '7'],
+        "platforms": {"SRF", "CWF"}
+    },
+    "FRED": {
+        "cpuid": ['7', '0', '1', '0', 'a', '17'],
+        "platforms": {}
```

Comment:
What does the empty of platforms mean? Seems FRED only be supported on CWF and DMR.